requirements.txt

```diff
@@ -0,0 +1,2 @@
+argparse; python_version < '2.7'
+# ovirt-engine-sdk >= 3.6  # It is not available in pip repo
```

Comment:
Apparently there is 3.6 version available: https://pypi.python.org/pypi/ovirt-engine-sdk-python/3.6.8.0